.github/workflows/tests.yml

```diff
@@ -144,6 +146,9 @@ jobs:
         with:
           name: cruby-gem
           path: gems
+      - name: Update pkgconf
+        if: env.MATRIX_RUNS_ON == 'windows-2019' || env.MATRIX_RUNS_ON == 'windows-2022'
+        run: pacman -Sy --noconfirm pkgconf
```

Comment:
https://github.com/mudge/re2/actions/runs/6980696994/job/18996462142 shows that v2.0.2 is being used. Watching https://packages.msys2.org/base/pkgconf for the update.